Edits to the license field signature verification topic (#1385)
replicatedhq/replicated-docs: docs/vendor/licenses-verify-fields-sdk-api.md

## Use Your Public Key to Verify License Field Signatures

In your application, you can use your public key (available in the vendor portal) and the MD5 hash of a license field value to verify the PSS signature of the license field.

To use your public key to verify license field signatures:

1. In the [vendor portal](https://vendor.replicated.com), go to the **Settings** page.

1. Click the **Replicated SDK Signature Verification** tab.

   ![signature verification page](/images/signature-verification.png)
   [View a larger version of this image](/images/signature-verification.png)

1. Under **Your public key**, copy the key and save it in a secure location.

1. (Optional) Under **Verification**, select the tab for the necessary programming language, and copy the code sample provided.

1. In your application, add logic that uses the public key to verify the integrity of license field signatures. If you copied one of the code samples from the vendor portal in the previous step, paste it into your application and make any additional edits as required. 

  If you are not using one of the code samples provided, consider the following requirements for verifying license field values:  

>     * License field signatures included in the response from the SDK API `/license/fields` and `/license/fields/{field-name}` endpoints are Base64 encoded and must be decoded before being decrypted.

Reviewer: ```suggestion
    * License field signatures included in the response from the SDK API `/license/fields` and `/license/fields/{field-name}` endpoints are Base64 encoded and must be decoded before they are verified.
```

>     * The MD5 hash of the license field value is required to verify the signature of the license field. The raw license filed value included in the response from the SDK API `/license/fields` and `/license/fields/{field-name}` endpoints must be encoded into an MD5 hash before it can be used for signature verification.

Reviewer: ```suggestion
    * The MD5 hash of the license field value is required to verify the signature of the license field. The raw license field value is included in the response from the SDK API `/license/fields` and `/license/fields/{field-name}` endpoints. The MD5 hash of the value must be calculated and used for signature verification.
```
